frontend/view-edit-client/client-section.component.tsx

```diff
@@ -5,14 +5,23 @@ import AuditSummarySection from "./audit-summary-section.component";
 import { LastUpdate } from "./view-client.component";
 
 export default function ClientSection(props: ClientSectionProps) {
-  const [expanded, setExpanded] = React.useState(true);
+  const [expanded, setExpanded] = React.useState(
+    JSON.parse(localStorage.getItem(props.title)) === null
```

Comment:
👍 nice and simple implementation. Let's use a prefix for the local storage key, so it doesn't get confused with any other local storage values that are set now or in the future.

Maybe `cu-clent-section-expanded:Demographics information`?